c/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
-cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+# For cmake_path function
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
```

Comment:
RHEL 7 is still in extended support. The next upgrade would be `3.11` as that's what RHEL 8 ships with.
So unless the current CMake version deprecates compatibility with v3.9 (as with v3.5), I'm unwilling to raise this requirement.

Reply:
I can use [this module](https://github.com/jtojnar/cmake-snips/blob/master/CMakeScripts/JoinPaths.cmake) instead. Is that ok?

Reply:
Sure, but please embed the function (and the copyright notice, the license is compatible) in the `CMakeLists.txt`.
Furthermore I'd request the new variables to be written in all-caps and maybe renamed to `PKG_CONFIG_INSTALL_*` mirroring the `GNUInstallDirs` naming scheme.

Reply:
Done